MekHQ/src/mekhq/gui/dialog/ResolveScenarioWizardDialog.java

```diff
@@ -340,165 +342,150 @@ private void initComponents() {
+        gridy = 2;
+
         Hashtable<Integer, JLabel> labelTable = new Hashtable<>();
         labelTable.put(0, new JLabel("0") );
         labelTable.put(1, new JLabel("1") );
         labelTable.put(2, new JLabel("2") );
         labelTable.put(3, new JLabel("3") );
         labelTable.put(4, new JLabel("4") );
         labelTable.put(5, new JLabel("5") );
-        j = 0;
+
         for (PersonStatus status : tracker.getSortedPeople()) {
-            j++;
             pstatuses.add(status);
-            nameLbl = new JLabel("<html>" + status.getName() + "<br><i> " + status.getUnitName() + "</i></html>");
-            miaCheck = new JCheckBox("");
-            miaBtns.add(miaCheck);
-            kiaCheck = new JCheckBox("");
-            kiaBtns.add(kiaCheck);
 
-            hitSlider = new JSlider(JSlider.HORIZONTAL, 0, 5, Math.min(status.getHits(),5));
-            hitSlider.setMajorTickSpacing(1);
-            hitSlider.setPaintTicks(true);
-            hitSlider.setLabelTable(labelTable);
-            hitSlider.setPaintLabels(true);
-            hitSlider.setSnapToTicks(true);
-            hitSliders.add(hitSlider);
-            if (status.isMissing()) {
-                miaCheck.setSelected(true);
-            }
-
-            kiaCheck.addItemListener(new CheckBoxKIAListener(hitSlider, miaCheck, null));
-            if (status.isDead()) {
-                kiaCheck.setSelected(true);
-            }
+            gridx = 0;
 
             gridBagConstraints = new GridBagConstraints();
-            gridBagConstraints.gridx = 0;
-            gridBagConstraints.gridy = i;
+            gridBagConstraints.gridx = gridx++;
+            gridBagConstraints.gridy = gridy++;
             gridBagConstraints.gridwidth = 1;
             gridBagConstraints.anchor = GridBagConstraints.NORTHWEST;
             gridBagConstraints.insets = new Insets(5, 5, 0, 0);
             gridBagConstraints.weightx = 0.0;
-            if (j == tracker.getPeopleStatus().keySet().size()) {
+            if (++j == tracker.getPeopleStatus().keySet().size()) {
                 gridBagConstraints.weighty = 1.0;
             }
+
+            nameLbl = new JLabel("<html>" + status.getName() + "<br><i> " + status.getUnitName() + "</i></html>");
             pnlPilotStatus.add(nameLbl, gridBagConstraints);
-            gridBagConstraints.gridx = 1;
+
+            JSlider hitSlider = new JSlider(JSlider.HORIZONTAL, 0, 5, Math.min(status.getHits(), 5));
+            hitSlider.setName(Integer.toString(j - 1));
+            hitSlider.setMajorTickSpacing(1);
+            hitSlider.setPaintTicks(true);
+            hitSlider.setLabelTable(labelTable);
+            hitSlider.setPaintLabels(true);
+            hitSlider.setSnapToTicks(true);
+            hitSliders.add(hitSlider);
+            gridBagConstraints.gridx = gridx++;
             pnlPilotStatus.add(hitSlider, gridBagConstraints);
-            gridBagConstraints.gridx = 2;
+
+            JCheckBox miaCheck = new JCheckBox("");
```

Comment:
Can you add accessibility text to describe this as a check box for MIA?

Reply:
By these, do you mean adding `miaCheck.setName("MIA");`?

Reply:
Roughly, or something from https://www.ibm.com/able/guidelines/java/snsjavagaccapi.html